apache-2.0.txt

```diff
@@ -1,202 +0,0 @@
-
```

Comment:
As has been noted internally, removing this file broke the import of Mbed TLS into Mbed OS. We should avoid such changes in an LTS branch unless there's a compelling reason. I am not aware of any legal reason why the license text itself should be in the file called `LICENSE`, rather than a reference to the license. Therefore I do not think there is any justification to making this change in LTS branches.

Reply:
OK, I'll revert this part of the change on the LTS branches